lib/fluent/plugin/kinesis_helper/client.rb

```diff
@@ -63,6 +63,10 @@ module ClientParams
             desc "Profile name. Default to 'default' or ENV['AWS_PROFILE']"
             config_param :profile_name, :string, default: nil
           end
+          config_section :process_credentials, multi: false do
+            desc "External process to execute."
+            config_param :process, :string, default: nil
```

Comment:
This config seems mandatory to use `process_credentials`. Is it ok to strip `default: nil`?